MediaBrowser.Controller/Entities/Folder.cs

```diff
@@ -333,8 +333,13 @@ protected virtual async Task ValidateChildrenInternal(IProgress<double> progress
             }
         }
 
-        private static bool IsLibraryFolderAccessible(IDirectoryService directoryService, BaseItem item)
+        private static bool IsLibraryFolderAccessible(IDirectoryService directoryService, BaseItem item, bool checkCollection)
         {
+            if (!checkCollection && (item is BoxSet || string.Equals(item.FileNameWithoutExtension, "collections", StringComparison.OrdinalIgnoreCase)))
```

Comment:
The folder name may be localized, see

https://github.com/jellyfin/jellyfin/blob/169698619dfe2989d0562b9723d0a01cfa02b9ba/Emby.Server.Implementations/Collections/CollectionManager.cs#L103